logstash.go

```diff
@@ -39,41 +68,61 @@ func NewLogstashAdapter(route *router.Route) (router.LogAdapter, error) {
 
 // Stream implements the router.LogAdapter interface.
 func (a *LogstashAdapter) Stream(logstream chan *router.Message) {
+
+	strTags := getopt("LOGSTASH_TAGS", "")
+	tags := strToSlice(strTags, ",")
+
 	for m := range logstream {
+
 		dockerInfo := DockerInfo{
 			Name:     m.Container.Name,
 			ID:       m.Container.ID,
 			Image:    m.Container.Config.Image,
 			Hostname: m.Container.Config.Hostname,
 		}
+		debug("logstash.Stream(): dockerInfo = Name:", m.Container.Name)
```

Comment:
As there only really is one method here, better to format logs as `logstash: ...`, skipping all () and method names.